src/components/content-tab-overview.tsx

```diff
@@ -225,24 +227,53 @@ export function ContentTabOverview( { selectedSite }: ContentTabOverviewProps )
 	} = useThemeDetails();
 
 	const loading = loadingThemeDetails || loadingThumbnails || initialLoading;
+	const siteRunning = selectedSite.running;
 
 	return (
 		<div className="p-8 flex max-w-3xl">
 			<div className="w-52 ltr:mr-8 rtl:ml-8 flex-col justify-start items-start gap-8">
 				<h2 className="mb-3 a8c-subtitle-small">{ __( 'Theme' ) }</h2>
 				<div
 					className={ cx(
-						'w-full min-h-40 max-h-60 overflow-hidden rounded-sm border border-a8c-gray-5 bg-a8c-gray-0 mb-2 flex items-center justify-center',
-						loading && skeletonBg,
-						isThumbnailError && 'border-none'
+						'w-full min-h-40 max-h-64 rounded-sm border border-a8c-gray-5 bg-a8c-gray-0 mb-2 flex justify-center',
+						loading && `h-64 ${ skeletonBg }`,
+						isThumbnailError && 'border-none',
+						siteRunning && 'hover:border-a8c-blueberry duration-300'
 					) }
 				>
 					{ isThumbnailError && ! loading && (
-						<div className="flex items-center justify-center w-full h-full leading-5 text-a8c-gray-50">
+						<div className="flex items-center justify-center w-full h-64 leading-5 text-a8c-gray-50">
 							{ __( 'Preview unavailable' ) }
 						</div>
 					) }
-					{ ! loading && (
+					{ ! loading && siteRunning && (
+						<button
+							aria-label={ __( 'Open site' ) }
+							className={ cx( `relative group focus-visible:outline-a8c-blueberry` ) }
+							onClick={ () => getIpcApi().openSiteURL( selectedSite.id ) }
+						>
+							<div
+								className={ cx(
+									`opacity-0 group-hover:opacity-90 group-hover:bg-white duration-300 absolute size-full flex justify-center items-center bg-white text-a8c-blueberry`
+								) }
```

Comment:
If we are not concatenating multiple class names, I'd suggest directly using a string value here instead of cx function:

```suggestion
								className={
									'opacity-0 group-hover:opacity-90 group-hover:bg-white duration-300 absolute size-full flex justify-center items-center bg-white text-a8c-blueberry'
								}
```

Reply:
Removed: https://github.com/Automattic/studio/pull/357/commits/ba3af505bd6dca48f19b9711b9b59ada214e1660